api/assets/emails/html/storage-rack.html

```diff
@@ -13,7 +13,7 @@ <h2>Dear <?php echo $data['LCRET'] ?></h2>
 				</ul>
 			<?php } ?>
 			<p>Please either fill out the dewar dispatch form if you would like your dewar to be returned to you:<br /><a href="https://ispyb.diamond.ac.uk/dewars/dispatch/<?php echo $data['DEWARID'] ?>">https://ispyb.diamond.ac.uk/dewars/dispatch/<?php echo $data['DEWARID'] ?></a></p>
-			<p>Or fill out the dewar transfer form if you would like it to be transferred to another beamline for another visit:<br /><a href="https://ispyb.diamond.ac.uk/dewars/transfer/<?php echo $data['DEWARID'] ?>">https://ispyb.diamond.ac.uk/dewars/transfer/<?php echo $data['DEWARID'] ?></a></p>
+			<p>Or fill out the dewar transfer form if you require more data collection time:<br /><a href="https://ispyb.diamond.ac.uk/dewars/transfer/<?php echo $data['DEWARID'] ?>">https://ispyb.diamond.ac.uk/dewars/transfer/<?php echo $data['DEWARID'] ?></a></p>
```

Comment:
Should we keep parts of the old message mentioning transfers can be performed through this page? Something among the lines of "[...] if you require more data collection time, or if you would like it to be transferred to another beamline for another visit"

Reply:
Originally a dewar transfer meant someone would physically carry the dewar to another beamline, whereas now they are all in the storage rack, and we are just allocating visits to dewars in the database. So I tried to keep the message general (even applicable to non-beamlines).

Reply:
That makes sense! Thanks for clarifying